python/pyarrow/tests/test_serialization.py

```diff
@@ -580,6 +580,21 @@ def deserialize_regex(serialized, q):
     p.join()
 
 
+def _get_modified_env_with_pythonpath():
+    # Prepend pyarrow root directory to PYTHONPATH
+    env = os.environ.copy()
+    existing_pythonpath = env.get('PYTHONPATH', '')
+    if sys.platform == 'win32':
+        sep = ';'
+    else:
+        sep = ':'
+
+    module_path, _ = os.path.split(pa.__path__[0])
```

Comment:
@pitrou is there a more approved way to get the module absolute directory path?

Reply:
`os.path.abspath(os.path.dirname(pa.__file__))`, no?